Sources_v3/Workers/UserUpdater.swift

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// Makes user-related calls to the backend and updates the local storage with the results.
+class UserUpdater<ExtraData: ExtraDataTypes>: Worker {
+    /// Mutes the user with the provided `userId`.
+    /// - Parameters:
+    ///   - userId: The user identifier.
+    ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func muteUser(_ userId: UserId, completion: ((Error?) -> Void)? = nil) {
+        apiClient.request(endpoint: .muteUser(userId)) {
+            completion?($0.error)
+        }
+    }
+    
+    /// Unmutes the user with the provided `userId`.
+    /// - Parameters:
+    ///   - userId: The user identifier.
+    ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func unmuteUser(_ userId: UserId, completion: ((Error?) -> Void)? = nil) {
+        apiClient.request(endpoint: .unmuteUser(userId)) {
+            completion?($0.error)
+        }
+    }
+    
+    /// Makes a single user query call to the backend and updates the local storage with the results.
+    ///
+    /// - Parameters:
+    ///   - userId: The user identifier
+    ///   - completion: Called when the API call is finished. Called with `Error` if the remote update fails.
+    ///
+    func loadUser(_ userId: UserId, completion: ((Error?) -> Void)? = nil) {
+        apiClient
+            .request(endpoint: .users(query: .user(withID: userId))) { (result: Result<UserListPayload<ExtraData.User>, Error>) in
+                switch result {
+                case let .success(payload):
+                    log.assert(
```

Comment:
Let's not use assert because this value comes from the server. A simple `log.error` should be enough here. We don't want the app to start crashing when something goes wrong on the server :)

Reply:
fixed in f703749d3bae7c64fff00849a4725c7ff49ce13c